lib/bless.ex

```diff
@@ -3,14 +3,29 @@ defmodule Bless do
   A mix task for running test suites
   """
 
+  # coveralls-ignore-start
   def default do
     [
       compile: ["--warnings-as-errors", "--force"],
       "coveralls.html": ["--raise"],
       format: ["--check-formatted"],
       credo: []
     ]
+    |> append_version_dependent_checks()
   end
-end
 
-# Generated by Elixir.Gaas.Generators.Simple.BaseModule
+  defp append_version_dependent_checks(checks) do
+    version_dependent_checks =
+      if Version.match?(System.version(), "~> 1.10") do
+        [
+          "deps.unlock": ["--check-unused"]
+        ]
+      else
+        []
+      end
```

Comment:
it might actually make sense to make this a `cond` in case we want to add more clauses

e.g.

```elixir
cond do
  Version.match?(System.version(), "~> 1.10") -> ["deps.unlock": ["--check-unused"]]
  # some hypothetical task in 1.12
  Version.match?(System.version(), "~> 1.12") -> ["do.cold.fusion": ["--nao-plox"]]
  true -> []
end
```